index.html

```diff
@@ -12,7 +12,9 @@
     <ul>
       <li><a href="tel:+79039059588" class="link"></a></li>
       <li><a href="" class="link"></a></li>
-      <li><a href="" class="link"></a></li>
```

Comment:
In vain developer deleted this line, have to fix a lot of mistake in other files.